src/main/java/com/revealprecision/revealserver/api/v1/dto/factory/FormDataEntityTagEventFactory.java

```diff
@@ -0,0 +1,32 @@
+package com.revealprecision.revealserver.api.v1.dto.factory;
+
+import com.revealprecision.revealserver.messaging.message.FormDataEntityTagEvent;
+import com.revealprecision.revealserver.messaging.message.FormDataEntityTagValueEvent;
+import com.revealprecision.revealserver.persistence.domain.Event;
+import com.revealprecision.revealserver.persistence.domain.Plan;
+import java.util.List;
+import java.util.UUID;
+
+public class FormDataEntityTagEventFactory {
+
+  // Proceed with caution here as new updates / removals to the object will prevent rewind of the streams application.
```

Comment:
maybe this comment should also be in the streams processing code or we make this code to not have that caution